README.md

```diff
@@ -1 +1,17 @@
-# insights
\ No newline at end of file
+# Insights
+
+### _add(insightObj, cb)_
+function to add a new insight, where `insightObj` takes the form:
+```js
+{
+  title: 'Insight Title',
+  url: 'www.ce100-insight-url.com',
+  doctype: '.pdf',
```

Comment:
What is doctype used for? 
![screen shot 2017-01-05 at 11 49 13](https://cloud.githubusercontent.com/assets/12638261/21680061/3701bd28-d33f-11e6-9fa3-c0b4c22bd8d7.png)

We also need insight-type which tells the user if it is an article/report/image...